### PR TITLE
fix: Lu editing improvement fixes

### PR DESCRIPTION
**Pull request by hatpick**
## Description

- Adds insert/tag menu to create trigger modal when default recognizer is luis
- Updates tooltip and help link for trigger phrases label when default recognizer is luis
- Prevents floating labeling menu where there are no Luis ML entities

## Task Item

#minor

### CODE REVIEW

**Comment by hatpick** [comment]
@tdurnford

File: Composer/packages/adaptive-form/src/components/fields/IntentField.tsx [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+  }
+
   const handleChange = () => {
     onChange(value);
   };
 
   return (
     <React.Fragment>
-      <FieldLabel description={description} helpLink={uiOptions?.helpLink} id={id} label={label} required={required} />
+      <FieldLabel description={description} helpLink={helpLink} id={id} label={label} required={required} />
```
Comment (tdurnford) [inline Composer/packages/adaptive-form/src/components/fields/IntentField.tsx:42]: Should you use uiOptions.helpLink when the recognizer isn't LUIS?

Comment (tdurnford) [reply Composer/packages/adaptive-form/src/components/fields/IntentField.tsx:42]: Nvm

**Review by tdurnford — APPROVED** [review]

File: Composer/packages/server/src/locales/en-US.json [hunk 2]
```diff
@@ -299,6 +299,9 @@
   "apr_9_2020_3c8b47d7": {
     "message": "Apr 9, 2020"
   },
+  "are_you_sure_you_want_to_delete_your_bot_this_acti_214a9e11": {
+    "message": "Are you sure you want to delete your bot? This action cannot be undone and your bot and all related files in the bot project folder will be permanently deleted. Your Azure resources will remain unchanged."
```
Comment (GeoffCoxMSFT) [inline Composer/packages/server/src/locales/en-US.json:303]: Super nit:  I would have 4 sentences here:
Are you sure you want to delete your bot? This action cannot be undone. Your bot and all related files in the bot project folder will be permanently deleted. Your Azure resources will remain unchanged.

**Review by GeoffCoxMSFT — APPROVED** [review]

File: Composer/packages/server/src/locales/en-US.json (hunk 2 above)
Comment (tdurnford) [reply Composer/packages/server/src/locales/en-US.json:303]: @GeoffCoxMSFT Out of curiosity, why would you opt for separate sentences?

**Pull request merged**